### ISSUE TITLE
decoding speed for transducer  model

### ISSUE THREAD

**Issue by xinqipony** [espnet/espnet#1728]
i tried to decode a transformer transducer model, it's very slow, any algo or plan we could speed it up?

**Comment by xinqipony** [comment]
anyone runs large set on the recog code, I think maybe there'e a bug in;
espnet\espnet\nets\pytorch_backend\transducer\transformer_decoder.py
ytu = torch.log_softmax(self.joint(hi, y[0]), dim=0)

should y[0] be y[0][0] here?
